content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -214,6 +215,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `SupportIPVSProxyMode`: Enable providing in-cluster service load balancing using IPVS.
   See [service proxies](/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies) for more details.
 - `SupportPodPidsLimit`: Enable the support to limiting PIDs in Pods.
+- `Sysctls`: Enable the sysctl support for tunning kernel parameters to better isolate resources of container.
```

Comment:
why "to better isolate resources of container." Don't see a connection to sysctls.